Dockerfile.build

```diff
@@ -1,15 +1,14 @@
 FROM golang:1.5
 
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
```

Comment:
Since the UI builds in its own build container, we can drop this node stuff from our build image.